src/points/cardDefinitions.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { PointsActivities, PointsCardMetadata } from 'src/points/types'
+import Celebration from 'src/icons/Celebration'
+import SwapArrows from 'src/icons/SwapArrows'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import Rocket from 'src/icons/Rocket'
+
+const cardDefinitions: Record<PointsActivities, PointsCardMetadata> = {
+  'create-wallet': {
+    title: 'points.activityCards.createWallet.title',
+    icon: <Celebration />,
+    defaultCompletionStatus: true,
+  },
+  swap: {
+    title: 'points.activityCards.swap.title',
+    icon: <SwapArrows />,
+    defaultCompletionStatus: false,
+    bottomSheet: {
+      title: 'points.activityCards.swap.bottomSheet.title',
```

Comment:
idk how serious we are about doing static extraction (we've been talking about it since i first joined and haven't really made a start on it, i'm also not sure it will work with OTA translations because once unused keys are removed, they might also be removed in Crowdin?) i saw @jeanregisser comment on this in another part of the PR, so we're not dynamically constructing the keys to keep the door for this open but i think the requirement is then also that we need to pass the `t` function in here?